src/app/contexts/ToggleContext/index.jsx

```diff
@@ -72,8 +29,16 @@ const ToggleContextConsumer = ToggleContext.Consumer;
 
 ToggleContextProvider.propTypes = {
   children: node.isRequired,
-  origin: string.isRequired,
-  service: string.isRequired,
+  remoteToggles: shape({
+    service: string.isRequired,
+    ads: shape({
```

Comment:
Does it make sense to hardcode the `ads` toggle here?

Reply:
Removed the ads key, we probably don't want to maintain this every time a new toggle is added. Or we're likely to forget anyway!